mods/ra/weapons/explosions.yaml

```diff
@@ -243,34 +243,34 @@ CrateNuke:
 	Warhead@3Eff_impact: CreateEffect
 		Explosions: nuke
 		ImpactSounds: kaboom1.aud
-	Warhead@4Dam_areanuke: SpreadDamage
+	Warhead@4Dam_areanuke1: SpreadDamage
```

Comment:
I'm considering (not entirely sure yet, due to the difference in total warhead count and relatively low yaml line savings) to make MiniNuke inherit CrateNuke, so I renamed this now so I won't have to touch CrateNuke if/when I do that.